org.eclipse.bluesky/language-configurations/scss/language-configuration.json

```diff
@@ -0,0 +1,25 @@
+{
```

Comment:
Please add comments like https://github.com/mickaelistria/eclipse-bluesky/blob/master/org.eclipse.bluesky/language-configurations/css/language-configuration.json#L1

Reply:
Hello I will fix this tomorrow.